packages/Liquid-Tests.package/LQPollTests.class/instance/testIDsAreUUIDs.st

```diff
@@ -0,0 +1,10 @@
+tests
+testIDsAreUUIDs	
+	"This test ensures that all ID's used in Liquid are UUID's to avoid collisions"
+	self assert: [LiquidAnswer new id isKindOf: UUID].
+	self assert: [LiquidChoicesAnswer new id isKindOf: UUID].
+	self assert: [LiquidAnswerSet new id isKindOf: UUID].
+	self assert: [LiquidPoll new id isKindOf: UUID].
+	self assert: [LiquidPollDraft new id isKindOf: UUID].
+	self assert: [LiquidQuestion new id isKindOf: UUID].
+	self assert: [LiquidChoicesQuestion new id isKindOf: UUID].
```

Comment:
while `isKindOf:` is considered a bad-practice, Squeak is missing an `isUUID` handler on `UUID` :(

Reply:
[dust] (maybe put the above in a comment)

Reply:
Exactly my thoughts, good point with the comment, I have included this in the test.